multistreaming-server/stunnel-conf/etc-stunnel-conf.d-ig.conf

```diff
@@ -0,0 +1,5 @@
+[fb-live]
```

Comment:
You need to give the Instagram stunnel config a different section name than the Facebook stunnel config. This is likely the reason that your tests show a failure in binding for `fb-live`. I would suggest the name here is `ig-live`.

Reply:
ugh... Sorry about that! Updated with that change. 

I'm still getting some errors though even after that change.

```
`[ ] Clients allowed=512000
[.] stunnel 5.56 on x86_64-alpine-linux-musl platform
[.] Compiled/running with OpenSSL 1.1.1d  10 Sep 2019
[.] Threading:PTHREAD Sockets:POLL,IPv6 TLS:ENGINE,OCSP,PSK,SNI
[ ] errno: (*__errno_location())
[.] Reading configuration from file /etc/stunnel/stunnel.conf
[.] UTF-8 byte order mark not detected
[ ] "/etc/stunnel/conf.d/." is not a file
[ ] "/etc/stunnel/conf.d/.." is not a file
[.] Reading configuration from file /etc/stunnel/conf.d/fb.conf
[.] UTF-8 byte order mark not detected
[ ] No PRNG seeding was required
[.] Reading configuration from file /etc/stunnel/conf.d/ig.conf
[.] UTF-8 byte order mark not detected
[ ] Initializing service [fb-live]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] No certificate or private key specified
[:] Service [fb-live] needs authentication to prevent MITM attacks
[ ] Initializing service [ig-live]
[ ] Ciphers: HIGH:!aNULL:!SSLv2:!DH:!kDHEPSK
[ ] TLSv1.3 ciphersuites: TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256
[ ] TLS options: 0x02100004 (+0x00000000, -0x00000000)
[ ] No certificate or private key specified
[:] Service [ig-live] needs authentication to prevent MITM attacks
[.] Configuration successful
[ ] Binding service [fb-live]
[ ] Listening file descriptor created (FD=9)
[ ] Setting accept socket options (FD=9)
[ ] Option SO_REUSEADDR set on accept socket
[.] Binding service [fb-live] to 127.0.0.1:19350: Address in use (98)
[!] Binding service [fb-live] failed
[ ] Deallocating section defaults
[ ] Unbinding service [fb-live]
[ ] Service [fb-live] closed
[ ] Deallocating section [fb-live]
[ ] Unbinding service [ig-live]
[ ] Service [ig-live] closed
[ ] Deallocating section [ig-live]`
```